bin/cli.js

```diff
@@ -195,6 +204,25 @@ function invoke(env) {
         .catch(exit);
     });
 
+  commander
+    .command('migrate:print')
```

Comment:
new command needs test. see https://github.com/tgriesser/knex/tree/master/test/jake for an example (https://github.com/tgriesser/knex/blob/master/test/jake/knexfile.js has a recommended structure, migrate.js will be cleaned up in the future)

Reply:
@kibertoad I did the integration tests: https://github.com/tgriesser/knex/pull/2932/files#diff-01d824a84fdd0b90ebaadc424247da50R334

I tried to create a test using the jake structure you are using, but I had some problems (listed below) and with integration tests the assert is better.

* in my local env the `bin_test` is not working I needed to run pointing to a file (ex: `jake -f test/jake/migrate.js`) to make it work
* the tests seem to run parallely and I need to remote the `db` created before to run the test using `print`, because my assert would be assert that the `db` was not created (what is a weak assertion)